pages/common/antibody.md

```diff
@@ -1,6 +1,6 @@
 # antibody
 
-> The fastest shell plugin manager.
+> "The fastest" shell plugin manager.
```

Comment:
What about a command description without a subjective descriptions?
```suggestion
> Shell plugin manager with focus in performance.
```

Reply:
@navarroaxel I think it should be `focus on performance`. Also, this PR is only for translations, should we really add this change here?